v1/suzu.json

```diff
@@ -77,6 +91,23 @@
           "to_state": "bootloader",
           "fallback_user_action": "bootloader"
         },
+        {
+          "type": "fastboot:flash",
+          "condition": {"var": "bootstrap", "value": true},
+          "flash": [
+            {
+              "partition": "oem",
+              "file": "SW_binaries_for_Xperia_AOSP_N_MR1_5.7_r1_v08_loire.img",
+              "group": "firmware",
+              "raw": true
+            }
+          ]
+        },
+        {
+          "type": "fastboot:erase",
+          "condition": {"var": "bootstrap", "value": true},
+          "partition": "system"
+        },
```

Comment:
Is this needed?

Reply:
If not, you might be able to merge the flashing steps

Reply:
It turns out it is required to wipe the system partition; this was one of the two missing pieces to get this sucker working.